### PR TITLE
Added article about application management with angular

### PR DESCRIPTION
**Pull request by tzraikov**

### CODE REVIEW

**Comment by tzraikov** [comment]
@tjvantoll

**Comment by hdeshev** [comment]
I think we should get rid of the JavaScript code in the article and use just TypeScript.

:+1: otherwise.

File: core-concepts/application-management-angular.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+
+* [Start Application](#start-application)
+* [Use Application Events](#use-application-events)
+* [Android Activity Events](#android-activity-events)
+* [iOS UIApplicationDelegate](#ios-uiapplicationdelegate)
+* [Persist and Restore Application Settings](#persist-and-restore-application-settings)
+
+## Modules 
+
+Angular applications are modular. A module is a file containing a block of code dedicated to a single purpose. It exports something of a value that can be used by other parts of the application. For example:
```
Comment (tjvantoll) [inline core-concepts/application-management-angular.md]: It exports a value that can...

File: core-concepts/application-management-angular.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+		this.name = "Angular!";
+	}
+}
+```
+
+## Component metadata
+
+The `@Component` decorator contains metadata describing how to create and present the component. Here are some of the configuration options:
+
+* **selector** - a CSS selector that tells Angular to create and insert instance of this component where it finds the selector in parent component's template. For example:
```
Comment (tjvantoll) [inline core-concepts/application-management-angular.md]: insert an instance

File: core-concepts/application-management-angular.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+* **template** - A visual tree that represents the component view. Here you can use all NativeScript UI elements and custom defined UI components.
+* **templateUrl** - The address of a file where the component template is located.
+* **styles** - CSS directives that define the component style.
+* **styleUrls** - An array containing URLs of CSS files that define the component style.
+* **animations** - The animations associated with this component.
+* **providers** - an array of dependency injection providers for services that the component requires.
+
+## Component lifecycle
+
+Component lifecycle is controlled by the Angular application. It creates, updates and destroys components. Lifecycle hooks are used to handle different events from the component lifecycle. Each hook method starts with the **ng** prefix. The following are some the component lifecycle methods:
```
Comment (tjvantoll) [inline core-concepts/application-management-angular.md]: The component lifecycle is...

**Comment by tjvantoll** [comment]
I made a few trivial grammatical comments, but other than that this looks really good. I’ll agree with @hdeshev that we should just remove the JavaScript examples.

One other thing to consider: since this article does duplicate a lot of the content from the “NativeScript Core” version of this article, you may want to put the content in the same file and use the `{% nativeScript %}` tag things.

**Pull request merged**